.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,27 @@
-_**Instruction Template**_
+# Issue Template
 
-**THIS IS AN EXAMPLE TEMPLATE**
+THIS IS AN EXAMPLE TEMPLATE
 
-**Make sure the following boxes are checked (and the placeholders replaced) *BEFORE* submitting this issue**
+Make sure the following boxes are checked (and the placeholders replaced) BEFORE submitting this issue.
 
-**Screenshots are more than welcome!**
+Screenshots are more than welcome!
 
-**Please remove any unused content (including these instructions) before submitting your issue.**
+Please remove any unused content (including these instructions) before submitting your issue.
 
-***IMPORTANT:* For folder icon requests please first see issue #650**
+IMPORTANT: For folder icon requests please first see issue: [650](https://github.com/vscode-icons/vscode-icons/issues/650)
 
-**Thanks :smile:**
+Thanks.
 
-**Checklist:**
+Note: Always remove this line and all above.
```

Comment:
this is already stated two lines before.

Reply:
You can never be certain that the user read through the entire instructions and understood them. Like apps, instructions need to be idiot proof.